blog/2020-09-angular-schematics-common-helpers.md

```diff
@@ -682,11 +685,139 @@ export function playground(_options: any): Rule {
 }
 ```
 
-If we now run `schematics ../playground/src/collection.json:ng-add --debug=false` from our example Angular project, we can see that the `ng add` schematic has called the `playground` schematic.
+If we now run `schematics ../playground/src/collection.json:ng-add --debug=false` from our example Angular project, we can see that the `ng-add` schematic has called the `playground` schematic.
+
+With this knowledge you can define small atomic Schematics that can be executed 'standalone' or from another schematic that combines multiple standalone Schematics and calls them with specific parameters.
+
+- [Check out the example in the playground repository on GitHub](https://github.com/d-koppenhagen/schematics-helpers-playground/tree/master/playground/src/ng-add)
+
+### Run Schematics by using the `schematic` and `externalSchematic` function
+
+Perfect, we can now execute and combine our Schematics.
```

Comment:
Consistent spelling of "schematics": upper or lowercase?

Reply:
Alsoways uppercase

Reply:
okay! :-) wasn't sure